### PR TITLE
feat: validate item thumbnail

### PR DESCRIPTION
**Pull request by ReidyT**
This PR include changes to validate the item's thumbnail.
I also remove unused actor in getUrl parameters.

### CODE REVIEW

**Review by pyphilia — APPROVED** [review]
Nice refactor! LGTM 🌃

File: src/services/item/plugins/publication/validation/moderators/itemValidationModerator.ts
Comment (Defferrard) [inline src/services/item/plugins/publication/validation/moderators/itemValidationModerator.ts:1]: You could have used `git mv` so we keep track of the modification of this file

**Review by Defferrard — APPROVED** [review]
LGTM ! Good job !

File: src/services/item/plugins/publication/validation/moderators/itemValidationModerator.ts
Comment (ReidyT) [reply src/services/item/plugins/publication/validation/moderators/itemValidationModerator.ts:1]: Yes you're right ! Some times VS code do it yourself but not always ^^

**Pull request merged**